### ISSUE TITLE
codes

### ISSUE THREAD

**Issue by ADITYA-THEE-CODER** [ADITYA-THEE-CODER/ML-#69]
import numpy as np
from sklearn.preprocessing import StandardScaler
from sklearn.neighbors import KNeighborsClassifier

# data
data = np.array([
    [20, 20000],
    [25, 30000],
    [30, 80000],
    [35, 120000]
])

labels = np.array([0, 0, 1, 1])

# -------------------
# WITHOUT SCALING
# -------------------
model1 = KNeighborsClassifier(n_neighbors=3)
model1.fit(data, labels)

pred1 = model1.predict([[28, 60000]])
print("Without Scaling:", pred1)

# -------------------
# WITH SCALING
# -------------------
scaler = StandardScaler()
scaled_data = scaler.fit_transform(data)

model2 = KNeighborsClassifier(n_neighbors=3)
model2.fit(scaled_data, labels)

# IMPORTANT: scale input too
new_point = scaler.transform([[28, 60000]])

pred2 = model2.predict(new_point)
print("With Scaling:", pred2)
